mobile-app/cypress/e2e/functional/wallet/auctions/auctions_search_filter.spec.ts

```diff
@@ -0,0 +1,376 @@
+    cy.getByTestID("batch_card_0_dTU10").should("not.exist");
+    cy.getByTestID("batch_card_0_dTD10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("not.exist");
+
+    // Check for search word DFI ETH
+    cy.getByTestID("auctions_search_input").clear().type("DFI ETH");
+    cy.getByTestID("search_title").contains("Search results for “DFI ETH”");
+    cy.getByTestID("batch_card_0_dTU10").should("not.exist");
+    cy.getByTestID("batch_card_0_dTD10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("not.exist");
+
+    // Check for reverse order search with word ETH DFI
+    cy.getByTestID("auctions_search_input").clear().type("ETH DFI");
+    cy.getByTestID("search_title").contains("Search results for “ETH DFI”");
+    cy.getByTestID("batch_card_0_dTU10").should("not.exist");
+    cy.getByTestID("batch_card_0_dTD10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("not.exist");
+
+    // Check for search word DUSD
+    cy.getByTestID("auctions_search_input").clear().type("DUSD");
+    cy.getByTestID("search_title").contains("Search results for “DUSD”");
+    cy.getByTestID("batch_card_0_dTU10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("not.exist");
+
+    // Check for search word BTC
+    cy.getByTestID("auctions_search_input").clear().type("BTC");
+    cy.getByTestID("search_title").contains("Search results for “BTC”");
+    cy.getByTestID("batch_card_0_dTU10").should("not.exist");
+    cy.getByTestID("batch_card_1_dTD10").should("not.exist");
+
+    // Check for result after clearing search
+    cy.getByTestID("auctions_search_input_clear_btn").click();
+    cy.getByTestID("search_title").contains(
+      "Search for auctions using collateral token names i.e. DFI DUSD dBTC."
+    );
+    cy.getByTestID("auctions_search_input_close").click();
+    cy.getByTestID("batch_card_0_dTU10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("exist");
+  });
+
+  it("should not able to filter auction", () => {
+    cy.getByTestID("bottom_tab_auctions").click();
+    // Check for least time left by default selection
+    cy.getByTestID("batch_card_0_dTU10").should("exist");
+    cy.getByTestID("batch_card_1_dTD10").should("exist");
+    cy.getByTestID("auctions_sorting_dropdown_arrow").click();
+    cy.getByTestID("select_sort_least_time_left_check").should("exist");
+    cy.getByTestID("select_sort_most_time_left_check").should("not.exist");
+
+    // Check for least time left by default selection
+    cy.getByTestID("select_sort_most_time_left").click();
+    cy.getByTestID("batch_card_0_dTD10").should("exist");
+    cy.getByTestID("batch_card_1_dTU10").should("exist");
+    cy.getByTestID("auctions_sorting_dropdown_arrow").click();
+    cy.getByTestID("select_sort_least_time_left_check").should("not.exist");
+    cy.getByTestID("select_sort_most_time_left_check").should("exist");
+  });
+});
+
+context("Wallet - Auctions Feature Gated", () => {
```

Comment:
curious why we move this to here from `auction.spec`?

Reply:
trying to reduce execution time for auction.specs e2e